lib/bigquery/dataset.js

```diff
@@ -177,6 +183,25 @@ Dataset.prototype.getMetadata = function(callback) {
  * dataset.getTables(function(err, tables, nextQuery, apiResponse) {
  *   // If `nextQuery` is non-null, there are more results to fetch.
  * });
+ *
+ * //-
+ * // Get the tables as a readable object stream.
+ * //-
+ * dataset.getTables()
+ *   .on('error', console.error)
+ *   .on('data', function(table) {})
```

Comment:
This is possibly taking nitpicks to a whole new level, but can you put in here: "table is a Table object.", in this style: https://github.com/GoogleCloudPlatform/gcloud-node/blob/master/lib/storage/bucket.js#L531